examples/simple-dg.py

```diff
@@ -329,21 +335,16 @@ def knl():
 
 # {{{ trace pair
 
-class TracePair:
-    def __init__(self, where, interior, exterior):
-        self.where = where
-        self.interior = interior
-        self.exterior = exterior
-
-    def __getitem__(self, index):
-        return TracePair(
-                self.where,
-                self.exterior[index],
-                self.interior[index])
```

Comment:
This was the dumbest bug that cost me an hour of my life. It was https://github.com/inducer/grudge/pull/15/commits/687a28bf850d168bb1109f2b78ebf3a844853feb all over again. Interestingly, the "bug" (DG blowing up) was caused by the new `TracePair` doing the right thing by construction.